expat/tests/xmltest.log.expected

```diff
@@ -7,4 +7,4 @@ Expected not well-formed: oasis/p06fail1.xml
 Expected not well-formed: oasis/p08fail1.xml
 Expected not well-formed: oasis/p08fail2.xml
 Passed: 1801
-Failed: 8
+Failed: 9
```

Comment:
If `Failed` count goes up by one for `tests/xmlconf/xmltest/valid/ext-sa/012.xml` _without_ `-H` why does `Passed` count not go up by one for the new additional test _with_ `-H`?